File: api/packages/v1/administrativo/services/tabelionato_protesto/go/tabelionato_protesto_dashboard_resumo_service.py
from packages.v1.administrativo.actions.tabelionato_protesto.tabelionato_protesto_dashboard_resumo_action import (
    TabelionatoProtestoDashboardResumoAction,
)


class TabelionatoProtestoDashboardResumoService:
    def execute(self):
        return TabelionatoProtestoDashboardResumoAction().execute()

File: app/src/packages/administrativo/services/TabelionatoProtestoDashboard/TabelionatoProtestoDashboardResumoService.ts
'use server';

import { TabelionatoProtestoDashboardResumoData } from '@/packages/administrativo/data/TabelionatoProtestoDashboard/TabelionatoProtestoDashboardResumoData';
import { withClientErrorHandler } from '@/shared/actions/withClientErrorHandler/withClientErrorHandler';

async function executeTabelionatoProtestoDashboardResumoService() {
  return await TabelionatoProtestoDashboardResumoData();
}

export const TabelionatoProtestoDashboardResumoService = withClientErrorHandler(
  executeTabelionatoProtestoDashboardResumoService,
);
